incubator/hnc/internal/reconcilers/object.go

```diff
@@ -382,6 +388,24 @@ func (r *ObjectReconciler) syncPropagated(ctx context.Context, log logr.Logger,
 	return actionNop, nil
 }
 
+func (r *ObjectReconciler) getSelector(ctx context.Context, log logr.Logger, inst *unstructured.Unstructured) labels.Selector {
+	annot := inst.GetAnnotations()
+	selectorstr, present := annot[selectorAnnot]
+	if present == false {
```

Comment:
Style: `!present`, not `present == false`

Reply:
done